ansible/roles/nginx/templates/rogeros_auth_proxy.conf.j2

```diff
@@ -22,7 +22,7 @@ server {
 }{% endif %}
 
 server {
-       listen 80{% if env_name == 'local' %} default_server{% endif %} ;
+       listen 80 {{ default_server|default('') }} ;
```

Comment:
Do we want to prefix these vars with the role name? I would hate to have a collision because someone else defined default_server